components/vf-summary/vf-summary.scss

```diff
@@ -92,6 +92,7 @@
 
 .vf-summary__title {
   @include set-type($vf-summary__title-typography);
+  font-weight: 400;
```

Comment:
@khawkins98 the more I look - the more this looks wrong. 

Can you remember the rationale - was it because the preceding heading loses significance slightly because they're similar?

Reply:
I think it was just design feedback that the news headings here looked "a bit too bold" at the original, so they were made 400: 

![image](https://user-images.githubusercontent.com/928100/106049964-0014b100-60e7-11eb-9b6b-ae3fed533a32.png)

Agree that it's a hack in the system. Not sure if the other typography changes might make it redundant